twin.go

```diff
@@ -96,13 +137,22 @@ func (s *Substrate) GetTwin(id uint32) (*Twin, error) {
 }
 
 // CreateTwin creates a twin
-func (s *Substrate) CreateTwin(identity Identity, ip net.IP) (uint32, error) {
+func (s *Substrate) CreateTwin(identity Identity, relay *string, pk []byte) (uint32, error) {
 	cl, meta, err := s.getClient()
 	if err != nil {
 		return 0, err
 	}
 
-	c, err := types.NewCall(meta, "TfgridModule.create_twin", ip.String())
+	relayOption := OptionRelay{}
+	pkOption := types.NewOptionBytesEmpty()
+	if relay != nil {
+		relayOption = OptionRelay{HasValue: true, AsValue: *relay}
+	}
+	if pk != nil {
```

Comment:
the code should return an error if pk == nil in the very beginning of the function

Reply:
Both PK and relay are optional values, no need to return an error if it's not passed